pkg/services/featuremgmt/service.go

```diff
@@ -18,16 +17,15 @@ var (
 	}, []string{"name"})
 )
 
-func ProvideManagerService(cfg *setting.Cfg, licensing licensing.Licensing) (*FeatureManager, error) {
```

Comment:
^^^ removes dependency on licensing

and avoids a circular dependency in enterprise that requires `db.DB` -- that also has feature flags as an input